tools/isobuild/meteor-npm.js

```diff
@@ -777,7 +777,7 @@ Profile("meteorNpm.runNpmCommand", function (args, cwd) {
           });
         }
       );
-    });
+    }).await();
```

Comment:
I still don't really get this; shouldn't 

```js
P1.then(() => P2).await()
```

and 

```js
P1.then(() => P2.await()).await()
```

Be functionally identical? I could imagine maybe an extra `setImmediate` or something but I would have thought it would yield at the same time and everything.